### PR TITLE
F2 F945 and F946 Tests

### PR DESCRIPTION
**Pull request by rahulGDS**
## Proposed changes
Implementing the test cases for US-945 and US-946

### What changed

Added test cases for new expiry date screen

### Why did it change

Happy and unhappy paths for below 3 documents types:
Non-UK passport
EU Driving Licence
EEA National ID Card

### Issue tracking

- [F2F-945](https://govukverify.atlassian.net/browse/F2F-945)

[F2F-945]: https://govukverify.atlassian.net/browse/F2F-945?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: test/browser/step_definitions/F2FDocumentSelectionNonUKPassport.step.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
-const { Given, When, Then } = require("@cucumber/cucumber");
+const { Given, When, Then, } = require("@cucumber/cucumber");
```
Comment (page-dan) [inline test/browser/step_definitions/F2FDocumentSelectionNonUKPassport.step.js:1]:
```suggestion
const { Given, When, Then } = require("@cucumber/cucumber");
```

File: y
Comment (page-dan) [inline y:1]: Please delete this file

File: test/browser/step_definitions/F2FDocumentSelectionNonUKPassport.step.js (hunk 1 above)
Comment (rahulGDS) [reply test/browser/step_definitions/F2FDocumentSelectionNonUKPassport.step.js:1]: done

File: test/browser/features/BrowserBasedTests/UnhappyPath/CMAChangeButton/F2FChangeAnswersCountryOfIssueChange.feature [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@mock-api:f2f-f2f-success @success @browser
+@mock-api:f2f-f2f-success @success @browser @test
```
Comment (page-dan) [inline test/browser/features/BrowserBasedTests/UnhappyPath/CMAChangeButton/F2FChangeAnswersCountryOfIssueChange.feature:1]:
```suggestion
@mock-api:f2f-f2f-success @success @browser
```

**Pull request merged**